packages/rtk-query-codegen-openapi/src/types.ts

```diff
@@ -53,14 +53,20 @@ export interface CommonOptions {
   hooks?: boolean;
 }
 
+export type TextMatcher = string | RegExp | (string | RegExp)[];
+
+export type EndpointMatchFunction = (operationName: string, operationDefinition: OperationDefinition) => boolean;
+
+export type EndpointMatcher = TextMatcher | EndpointMatchFunction;
```

Comment:
```suggestion
export type TextMatcher = string | RegExp | (string | RegExp)[];

export type EndpointMatcherFunction = (operationName: string, operationDefinition: OperationDefinition) => boolean;

export type EndpointMatcher = TextMatcher | EndpointMatcherFunction;
```

Just to keep our naming consistent - we typically refer to these as matcher functions throughout the code and docs.